src/test/java/org/broadinstitute/hellbender/utils/config/ConfigUnitTest.java

```diff
@@ -0,0 +1,277 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.Accessible;
+import org.aeonbits.owner.ConfigCache;
+import org.aeonbits.owner.ConfigFactory;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import static java.nio.file.StandardCopyOption.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Unit test for GATK configuration file handling.
+ *
+ * Created by jonn on 7/19/17.
+ */
+public class ConfigUnitTest extends BaseTest {
+
+    private final static String testResourceDir = publicTestDir + "org/broadinstitute/hellbender/utils/config/";
+
+    // ================================================================================
+    // Helper Methods:
+    // ================================================================================
+
+    void listAndStoreConfigToStdOut(Accessible config) throws IOException {
+
+        config.list(System.out);
+        System.out.println();
+
+        config.store(System.out, "");
+        System.out.println();
+
+        config.storeToXML(System.out, "");
+        System.out.println();
+    }
+
+    void validateAndClearProperty(String key, Object value) {
+
+        String other = null;
+
```

Comment:
Yeah - since whitespace is free I tend to use it to make things more readable.  I've inherited and had to maintain some pretty old / poorly written code, so this is now a habit.

I'll condense it up a bit.